web/src/navigator.rs

```diff
@@ -145,19 +145,19 @@ impl NavigatorBackend for WebNavigatorBackend {
         let window = window().expect("window()");
 
         if url.scheme() != "javascript" {
-            if self.open_url_mode == OpenURLMode::Confirm {
+            if self.open_links == OpenLinks::Ask {
                 let message = format!("The SWF file wants to open the website {}", &url);
                 // TODO: Add a checkbox with a GUI toolkit
-                let confirm = window
+                let ask = window
```

Comment:
I feel like the variable name `confirm` should be kept as-is (instead of changing it to `ask`), here and in other places.